components/PlantOwnedButton/index.js

```diff
@@ -0,0 +1,34 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function PlantOwnedButton({
+  isOwned,
+  handleToggleOwned,
+  plantId,
+}) {
+  return (
+    <StyledButton type="button" onClick={() => handleToggleOwned(plantId)}>
+      {isOwned === true ? (
+        <Image
+          src={"./heart-solid.svg"}
```

Comment:
Short hint: No need to use { } for the src attribute as long as you write plain text.